composer.json

```diff
@@ -34,11 +34,14 @@
         "cs-fix": "phpcbf --colors --standard=vendor/cakephp/cakephp-codesniffer/CakePHP src/"
     },
     "require": {
-        "cakephp/cakephp": "^4.0"
+        "php": ">=8.1",
```

Comment:
This is not necessary. Since we fully depend on the CakePHP, the framework defines the lower and upper PHP bounds.

Reply:
Change applied, thanks